internal/pkg/mount/mount.go

```diff
@@ -161,6 +162,7 @@ func mountRetry(f RetryFunc, p *Point, isUnmount bool) (err error) {
 
 // Point represents a Linux mount point.
 type Point struct {
+	sync.Mutex
```

Comment:
Embedding mutex adds Lock and Unlock methods to Point's exported method set. That's never a good idea. That should be a regular field

Reply:
What particular issues can embedding mutex Lock and Unlock cause?